src/webtransport.js

```diff
@@ -497,42 +669,77 @@ class Http3WTSession {
           throw new Error('Got bidirectional stream without asking for it')
         this.rejectBiDi.shift()
         const curres = this.resolveBiDi.shift()
-        curres(strobj)
+
+        if (curres != null && strobj.readable != null && strobj.writable != null) {
+          curres({
+            readable: strobj.readable,
+            writable: strobj.writable
+          })
+        }
       } else {
         if (this.resolveUniDi.length === 0)
           throw new Error('Got unidirectional stream without asking for it')
         this.rejectUniDi.shift()
         const curres = this.resolveUniDi.shift()
-        curres(strobj.writable)
+
+        if (curres != null && strobj.writable != null) {
+          /** @type {WebTransportSendStream} */
+          // @ts-expect-error `getStats` property is missing from WritableStream
+          // we add it on the next line
+          const sendStream = strobj.writable
+          sendStream.getStats = () => {
+            return Promise.resolve({
+              timestamp: 0,
+              bytesWritten: 0n,
+              bytesSent: 0n,
+              bytesAcknowledged: 0n
+            })
+          }
+
+          curres(sendStream)
+        }
       }
     }
   }
 
+  /**
+   * @param {DatagramReceivedEvent} args
+   */
   onDatagramReceived(args) {
     // console.log('datagram received', args.datagram, Date.now())
     this.incomDatagramController.enqueue(args.datagram)
   }
 
+  /**
+   * @param {DatagramSendEvent} args
+   */
   onDatagramSend(args) {
     if (this.state === 'closed') return
     this.writeDatagramRej.shift()
     this.writeDatagramProm.shift()
     const res = this.writeDatagramRes.shift()
-    res()
+
+    if (res != null) {
+      res()
+    }
   }
 
+  /**
+   * @param {SessionReadyEvent | SessionCloseEvent | DatagramReceivedEvent | DatagramSendEvent | NewStreamEvent} args
+   */
   static callback(args) {
+    console.log('Session callback called', args)
```

Comment:
Please comment, only for debugging while developing

Reply:
Ha, oops.  I think I added that for local debugging.

Reply:
I've removed this.

Reply:
Or at least, put it back to how it was..

Reply:
good..